In file included from /usr/include/string.h:630:0,
		 from libmpd-strfsong.c:29:
libmpd-internal.h:210:10: error: expected identifier or '(' before '__extension__'
	char *   strndup     (const char *s, size_t n);

--- a/src/libmpd-strfsong.c
+++ b/src/libmpd-strfsong.c
@@ -28,6 +28,7 @@
 #include <unistd.h>
 #include <string.h>
 #include <glib.h>
+#include "config.h"
 #include "libmpd.h"
 #include "libmpd-internal.h"
 
